src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const checkPartner = person.sex === 'm'
+    ? `${person.partnerName} is my wife`
+    : `${person.partnerName} is my husband`;
```

Comment:
```suggestion
  const personPartner = person.sex === 'm'
    ? `${person.partnerName} is my wife`
    : `${person.partnerName} is my husband`;
```

Reply:
checkPartner sounds like function but you have variable :)
Also as you can see difference is only in partner naming `wife/husband` so you can keep only this part in variable and other text you can keep in your jsx

Reply:
done, as much as I understand it!